java/examples/datasets/Makefile.am

```diff
@@ -27,7 +27,7 @@ classes:
 
 pkgpath = examples/datasets
 hdfjarfile = jar$(PACKAGE_TARNAME)-$(PACKAGE_VERSION).jar
-CLASSPATH_ENV=CLASSPATH=.:$(JAVAROOT):$(top_builddir)/java/src/$(hdfjarfile):$(top_srcdir)/java/lib/slf4j-api-1.7.25.jar:$(top_srcdir)/java/lib/ext/slf4j-simple-1.7.25.jar:$$CLASSPATH
```

Comment:
I can't help but thinkthis should be set at the top level somewhere so we don't have to update a bunch of Makefile.am files when the libs change.